classes/clonecategory_form.php

```diff
@@ -33,44 +26,44 @@
  * @copyright 2018 tim@avide.com.au
  * @license http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
-class local_clonecategory_form extends moodleform {
+class clonecategory_form extends \moodleform {
```

Comment:
tiny nit, forms can go, but don't need to go into the own 2nd level namespace

classes/clonecategory_form.php

classes/form/clonecategory_form.php